src/styles/variables.scss

```diff
@@ -0,0 +1,9 @@
+$minCardWidth: 100px;
+$maxCardWidth: 200px;
+$starCount: 4;
+
+:root {
+  --dark-gray: #060b35;
+  --light-gray: #616070;
+  --light-blue: #00acdc;
+}
```

Comment:
Now you are combining css and scss variables in one file, this is not a very good practice, try to use only scss variables in this case